### DISCUSSION TITLE
RuntimeError: Unable to find a valid cuDNN algorithm to run convolution

### DISCUSSION THREAD

**Discussion by tacom1** [Project-MONAI/MONAILabel#1023] [Q&A]
I am trying to config MONAI Label in my laptop, but some errors occur. 

I don't known whether is related to GPU Memory(https://github.com/Project-MONAI/MONAILabel/discussions/943), because it's only cost  973MiB / 4GB(RTX 2050) when i click Run(deepedit_seg) in OHIF with sample data. 

Then i try to config in NVIDIA TITAN XP(12 GB), error is disapper(cost only 2.5 GB memory, which make me confused). Both computers run on Ubuntu 20.04, with cuda 11.3 and  Driver Version: 515.65.01.

Maybe i need to upgrade my device, can you give me some advise?

**Comment by SachidanandAlle** [comment]
4GB(RTX 2050) is too low.. if you are looking for something solid.. atleast go with 16 GB.. so that you can run some experiments and achieve better user experience while running infer/training..
